Sources/TextView/TextView.swift

```diff
@@ -95,7 +95,9 @@ public struct TextView: View {
 		
 		public func updateUIView(_ textView: UITextView, context _: Context) {
 			if !shouldWaitUntilCommit || textView.markedTextRange == nil {
+				let selectedRange = textView.selectedRange
 				textView.text = text
+				textView.selectedRange = selectedRange
```

Comment:
Curious, is the assumption here that any changes to the text occurred after the cursor position? Otherwise, let's say the text got a bunch shorter, wouldn't this range be out of bounds?